app/pages/services/servicesPageController.ts

```diff
@@ -25,21 +28,28 @@ export class ServicesPageController {
   };
 
   public update() {
+
+    // TODO: Remove these try/catches once we are not relying on 'additionalServers' in config.js
+
     try {
       this.dataService.list({
         group: 'servicecatalog.k8s.io',
         resource: 'serviceclasses'
       }, {}, (resources: any) => {
-        this.ctrl.services = resources.by("metadata.name");
-        this.ctrl.categories = this.constants.SERVICE_CATALOG_CATEGORIES;
-        this.ctrl.loading = false;
+        this.ctrl.serviceClasses = resources.by("metadata.name");
+      });
```

Comment:
@dtaylor113 Your calls to `dataService.list` should use the promise returned rather than passing in the callback as a parameter.